test/controllers/api/base_test_case.rb

```diff
@@ -7,7 +7,7 @@ def self.guard_incorrect_token!(path, args: 0, method: :get)
         url = send(path, *Array.new(args, 'a')) # rubocop:disable Lint/RedundantSplatExpansion
         send(method, url, as: :json)
 
-        assert_response 401
+        assert_response :unauthorized
```

Comment:
I noticed that the existing code used both `assert_response 200` and `assert_response :success`. I looked into using the keyword variant a bit, and quite liked it. Let me know if you like the change or would like me to revert.

Reply:
This is good! :)